src/components/Footer/components/index.ts

```diff
@@ -0,0 +1,173 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import { media } from 'components/breakpoints';
+import { colors, GU } from 'components/theme';
+
+// MAIN CONTAINERS
```

Comment:
Let's remove this comment :).

Reply:
Fixed in bf544b3bed8d0c40e101bd75c3b0adaca5f84567